tests/python/tir-base/test_tir_renew_defs.py

```diff
@@ -82,7 +82,8 @@ def _get_block(f):
 
 
 def test_match_buffer():
-    @T.prim_func
+    # well-formed checker complains about multiple definitions for a variable A0_s1>?
```

Comment:
Not sure what was going on here, but the well-formed checker complains. Bug or intended?

Reply:
I'd say bug in the unit test.  The `A0_s1` is a variable generated to represent `A.strides[1]`.  The `strides = [s, s]` should probably be `strides = [s, 1]`.

Reply:
Switching the strides value causes the test to fail later (the assert `matched_buffer1.strides[1] != matched_buffer2.strides[1]` fails), so I don't think that's the issue. I do not know what the intent of the test case is, but it may be necessary to make bigger changes to it.